operations/mimir-mixin/dashboards/queries.libsonnet

```diff
@@ -227,11 +227,8 @@ local filename = 'mimir-queries.json';
         $.panel('Data fetched / sec') +
         $.queryPanel(|||
           sum by(data_type) (
-            # non-streaming store-gateway will not have the stage label on any fetched metrics
-            rate(cortex_bucket_store_series_data_size_fetched_bytes_sum{component="store-gateway", stage="", %(jobMatcher)s}[$__rate_interval])
-            or
-            # streaming store-gateway with new caching will have overlapping metrics for fetched chunks data; we select only the most narrow one - "fetched"
-            rate(cortex_bucket_store_series_data_size_fetched_bytes_sum{component="store-gateway", data_type="chunks", stage="fetched", %(jobMatcher)s}[$__rate_interval])
+            # Exclude "chunks refetched".
```

Comment:
Note to reviewers: logically this should be the same exact query as before (manually tested too).